client/my-sites/site-settings/form-writing.jsx

```diff
@@ -104,7 +104,7 @@ class SiteSettingsFormWriting extends Component {
 
 							<CustomContentTypes
 								onSubmitForm={ this.props.handleSubmitForm }
-								handleToggle={ handleToggle }
+								handleToggle={ handleAutosavingToggle }
```

Comment:
Haha, yeah, you're right about this one. I just probably prefer us to be accurate about it, I don't really like renaming props in the component flow, because it can be confusing to folks who're not familiar with the code.